public/css/style.css

```
@@ -163,3 +163,124 @@
+    flex-shrink: 0;
+}
+
+.carousel-images {
+    left: 5px;
+    width: 100%;
+    height: 100%;
+    position: relative;
+    border-radius: 50%;
+    overflow: hidden;
+    border: 0.5rem;
+    box-shadow: 0 0 30px rgba(34, 197, 94, 0.3);
```

Stylelint (17.2.0)

[error] Expected "0.3" to be "30%" (alpha-value-notation)

(alpha-value-notation)